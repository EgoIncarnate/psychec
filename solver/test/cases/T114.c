void f()
{
    static struct option const long_options[] =
    {
      {"number-nonblank", no_argument, NULL, 'b'},
    };
}
