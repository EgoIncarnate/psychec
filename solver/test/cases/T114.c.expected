typedef  struct option {char * member_0;char member_3;int /*<<< orphan or scalar*/* member_2;int /*<<< orphan or scalar*/ member_1;}  option ;
#define  no_argument 128
